regression/regression_test_enkf.sh

```diff
@@ -321,16 +355,35 @@ else
 
 # Next, check reproducibility of results between exp1 and exp3
 
-   if [[ `expr substr $exp1 1 4` = "rtma" ]]; then
+   if [[ `expr substr $exp1 1 4` = "rrfs" ]]; then
```

Comment:
`"${exp1:0:4}" == "rrfs"`

Reply:
the rtma test case will be replaced by rrfs, right?

Reply:
Yes, rtma does not have enkf run. This is for check enkf results only.

Reply:
done